app/javascript/pages/account/account-settings.tsx

```diff
@@ -258,10 +341,33 @@ const AccountSettings = ({ profile }: { profile: User }) => {
         <div className="flex flex-wrap relative md:max-w-lg mx-auto md:py-8">
           <Card className="w-full pb-8">
             <AccountSettingsHeader />
-            <NameSection user={user} setUser={setUser} />
-            <DateOfBirthSection user={user} setUser={setUser} />
-            <EmailSection user={user} setUser={setUser} />
-            <PasswordSection user={user} setUser={setUser} />
+            {banners.nameUpdateBanner && (
+              <span className="mb-8">
+                <UpdateBanner />
+              </span>
+            )}
+            {banners.nameSavedBanner && <SavedBanner />}
+            <NameSection user={user} setUser={setUser} handleBanners={handleBanners} />
```

Comment:
I wonder if we should pass the banners as a prop to their respective sections so that it is clearer which banner goes with each section?

Reply:
```suggestion
            <NameSection user={user} setUser={setUser} handleBanners={handleBanners} banners={<>
            {banners.nameUpdateBanner && (
              <span className="mb-8">
                <UpdateBanner />
              </span>
            )}
            {banners.nameSavedBanner && <SavedBanner />}
            </>} />
```

or even

```suggestion
            <NameSection user={user} setUser={setUser} handleBanners={handleBanners} banners={banners} />
```

Or something like that. What do you think?

Reply:
Ah, good point! Okay, that makes sense